client/reader/stream/index.jsx

```diff
@@ -4,7 +4,7 @@
 import ReactDom from 'react-dom';
 import React, { PropTypes } from 'react';
 import classnames from 'classnames';
-import { defer, findLast, flatMap, noop, times, clamp, includes, last } from 'lodash';
+import { defer, findLast, noop, times, clamp, includes, } from 'lodash';
```

Comment:
I see the point of adding a trailing comma for multi-line, but not so sure here.